### PR TITLE
new UX to configure java runtime

### PR DESCRIPTION
**Pull request by Eskibear**

### CODE REVIEW

**Comment by Eskibear** [comment]
Now:
image


I'll create a new PR to add Install JDK webview, which looks like below shown aside:
image

File: src/java-runtime/assets/ProjectJDKPanel.tsx [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+      .map((p, index) => (
+        <DataRow key={index}>
+          <DataCell gridColumn="1">{p.name}</DataCell>
+          <DataCell gridColumn="2"><div className="inline-flex">{p.projectType}{projectTypeHint(p.projectType)}</div></DataCell>
+          <DataCell gridColumn="3">
+            {
+              hasBuildTool(p) ?
+                <div className="inline-flex">
+                  <span>{p.sourceLevel}</span>
+                  <Button appearance="icon" onClick={() => this.onClickEdit(p)}><span className="codicon codicon-edit"></span></Button>
```
Comment (jdneo) [inline src/java-runtime/assets/ProjectJDKPanel.tsx:61]: We can have a tooltip for this button as well

**Comment by jdneo** [comment]
I tried to open runtime page for petclinic and see this:

image

**Review by jdneo — APPROVED** [review]

**Comment by Eskibear** [comment]
futher actions tracked in  #798

**Pull request merged**